source/_components/homekit.markdown

```diff
@@ -0,0 +1,63 @@
+---
+layout: page
+title: "HomeKit"
+description: "Instructions how to setup the HomeKit component in Home Assistant."
+date: 2018-02-20 17:30
+sidebar: true
+comments: false
+sharing: true
+footer: true
+ha_category: Voice
+ha_release: 0.64
+logo: apple-homekit.png
+---
+
+The `HomeKit` component allows you to forward entities from Home Assistant to Apple `HomeKit`, so they could be controlled from Apple `Home` app and `Siri`.
+
+{% configuration %}
+  homekit:
+    description: HomeKit configuration.
+    required: true
+    type: map
+    keys:
+      pincode:
+        description: Pin code required during setup of HomeKit Home Assistant accessory. The format needs to be 'XXX-XX-XXX' where X is a number between 1 and 9.
+        required: false
+        type: string
+        default: '"123-45-678"'
+      port:
+        description: Port for the HomeKit extension.
+        required: false
+        type: int
+        default: 51826
+{% endconfiguration %}
+
+## {% linkable_title Setup %}
+
+To enable the `HomeKit` component in Home Assistant, add the following to your configuration file:
+
+```yaml
+# Example for HomeKit setup
+homekit:
+  pincode: '123-45-678'
+```
+
+<p class='note'>It is not recommended to choose '123-45-678' as your pin code!</p>
+
+After Home Assistant has started, all supported entities (see the [list](#supported-components) below which types are already integrated) will be exposed to `HomeKit`. To add them:
```

Comment:
This should be `compoments` Instead of `types`. Types are just a way to organize the different adaptaions of a component.

Reply:
you are right, thanks. OK to merge after this?